context/datapath.go

```diff
@@ -480,6 +480,15 @@ func (dpNode *DataPathNode) CreateSessRuleQer(smContext *SMContext) (*QER, error
 
 //ActivateUpLinkPdr
 func (dpNode *DataPathNode) ActivateUpLinkPdr(smContext *SMContext, defQER *QER, defPrecedence uint32) error {
+
+	ueIpAddr := pfcpType.UEIPAddress{}
+	if dpNode.UPF.IsUpfSupportUeIpAddrAlloc() {
```

Comment:
How do you do pfcp signalling when multiple UPFs ? One after the other ? Please note that logically we should get UE address from UPF and send it to iUPF... Also we should set CHV4=true only for anchor UPF